frontends/web/src/routes/device/manage-backups/manage-backups.tsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ * Copyright 2024 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { TDevices } from '../../../api/devices';
+import { ButtonLink } from '../../../components/forms';
+import { Guide } from '../../../components/guide/guide';
+import { Entry } from '../../../components/guide/entry';
+import { Header } from '../../../components/layout';
+import { Backups } from '../bitbox01/backups';
+import { BackupsV2 } from '../bitbox02/backups';
+import { SDCardCheck } from '../bitbox02/sdcardcheck';
+
+type TProps = {
+  deviceID: string;
+  devices: TDevices
+}
+
+export const ManageBackups = ({
+  deviceID,
+  devices,
+}: TProps) => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+
+  if (!devices[deviceID]) {
+    navigate('/');
+    return null;
```

Comment:
nit: instead of `return null;` you could `return;` more concise and we seem to do it like in other places where we use `navigate(..)` as well.

Reply:
I think `return null` makes sense to be more explicit when we want to say "don't render this" e.g. [here](https://github.com/digitalbitbox/bitbox-wallet-app/pull/2706/files#diff-00e572a4a466c40766a05fbb1474a35a0efa21a505387b30425c9130b5f9dbc0R138). But not when _aborting_/_navigating_.

Reply:
I don't think navigate call is synchronous. If there is no device `!devices[deviceID]` this component should render null.